tests/testthat/_snaps/variables.md

```diff
@@ -0,0 +1,86 @@
+# variable() errors informatively
+
+    Code
+      variable(upper = NA)
+    Error <simpleError>
+      lower and upper must be numeric
```

Comment:
too verbose? should only report on upper?

Reply:
The error message I think captures both cases, where lower can be numeric and upper something else?

Reply:
could split the test in two? check each one separately and then the conflict between them? Though that's probably yak shaving. fine to leave it as is

Reply:
I've added a couple more tests for this - I think that covers it.